### ISSUE TITLE
"currentLayer.resizeRoot_ is not a function". Plugin ends with exception when I run the provided example.

### ISSUE THREAD

**Issue by frankgruber** [hrescak/Sketch-Flex-Layout#16]
Im using Sketch 3.5.2. After installing the plugin, going through the provided examples does not end with the expected results as the scripts "throws an exception".
From my experience this did not happen in older sketch versions.

```
Feb 19 10:18:04 MyComputersName Compute Layout (Sketch Plugin)[4928]: ======= DEBUG START =========
Feb 19 10:18:04 MyComputersName Compute Layout (Sketch Plugin)[4928]: 0 classes found in prototypes
Feb 19 10:18:04 MyComputersName Compute Layout (Sketch Plugin)[4928]: --13ms-- Stylesheet + prototypes parsed
Feb 19 10:18:04 MyComputersName Compute Layout (Sketch Plugin)[4928]: --7ms-- Applying stylesheet to layers
Feb 19 10:18:04 MyComputersName Compute Layout (Sketch Plugin)[4928]: --20ms-- Computed styles
Feb 19 10:18:04 MyComputersName Compute Layout (Sketch Plugin)[4928]: --4ms-- Measures collected
Feb 19 10:18:04 MyComputersName Compute Layout (Sketch Plugin)[4928]: --9ms-- Recomputed styles
Feb 19 10:18:04 MyComputersName Sketch[4928]: Exception: TypeError: currentLayer.resizeRoot_ is not a function. (In 'currentLayer.resizeRoot_( true)', 'currentLayer.resizeRoot_' is undefined)
Feb 19 10:18:04 MyComputersName Compute Layout (Sketch Plugin)[4928]: TypeError: currentLayer.resizeRoot_ is not a function. (In 'currentLayer.resizeRoot_( true)', 'currentLayer.resizeRoot_' is undefined)
    sourceURL: /Users/me/Library/Application Support/com.bohemiancoding.sketch3/Plugins/Flex-Layout.sketchplugin/Contents/Sketch/Layout/layout.js
```

![screen shot 2016-02-19 at 10 27 14](https://cloud.githubusercontent.com/assets/2472274/13171599/ea7f9fa8-d6f3-11e5-9be8-e29467dad6af.png)

**Comment by rabbit-roger** [comment]
I have the same problem.

**Comment by frankgruber** [comment]
http://developer.sketchapp.com/reference/MSLayerGroup/

**resizeRoot:**

Available in Sketch 3.4 and below, removed in Sketch 3.5 (see resizeToFitChildrenWithOption:). Resizes the group to fit around all of its sub-layers.

**resizeToFitChildrenWithOption:option**

Added in Sketch 3.5. Resizes the group to fit around all of its sub-layers. option is either 0 (MSLayerGroupResizeOptionLayerOnly) or 1 (MSLayerGroupResizeOptionResizeParentUponRectChange).
